src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
-export interface Figure {
+type ShapesType = 'triangle' | 'circle' | 'rectangle';
+type ColorsType = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape : ShapesType,
+  color: ColorsType,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape : ShapesType = 'triangle';
 
-}
+  constructor(
+    public color: ColorsType,
+    public a: number = 0,
+    public b: number = 0,
+    public c: number = 0,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c) {
+      throw new Error('Enter valid values');
+    }
```

Comment:
Btw, in all cases we have to check if passed value is greater than 0, let's create helper function for it and reuse in all cases

Reply:
`a + b <= c`
But what if we'll pass `1 3 2` to this function?